### PR TITLE
Updates Caddy config example to work

### PR DESCRIPTION
**Pull request by tankerkiller125**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -68,14 +68,13 @@ Caddy requires a very simple configuration in order for Flarum to work properly.
 ```
 www.example.com {
     root * /var/www/flarum/public
-    try_files {path} {path}/ /index.php
```
[the remark this answers is not in the document]
Comment (tankerkiller125) [reply docs/install.md:71]: It's no longer needed, according the authors of caddy the php_fastcgi directive now automatically includes the try_files built right in.

File: docs/install.md [hunk 2]
```diff
@@ -68,14 +68,13 @@ Caddy requires a very simple configuration in order for Flarum to work properly.
...
-    php_fastcgi / /var/run/php/php7.4-fpm.sock php
+    php_fastcgi unix//var/run/php/php7.4-fpm.sock
     header /assets {
         +Cache-Control "public, must-revalidate, proxy-revalidate"
         +Cache-Control "max-age=25000"
-        Pragma "public" 
+        Pragma "public"
```
[the remark this answers is not in the document]
Comment (tankerkiller125) [reply docs/install.md:75]: It looks like an extra unneeded space was removed.

[the remark this answers is not in the document]
Comment (askvortsov1) [reply docs/install.md:75]: Not sure how I missed that... :see_no_evil:

**Review by askvortsov1 — APPROVED** [review]

**Comment by ynyyn** [comment]
Still not got merged... Any progress? 
I am new to caddy (v2), so I followed the docs to setup, and as you may guess, I got confused why it did not work... 
If it had been merged earlier, it would save my day, or maybe many other's.

**Pull request merged**